### PR TITLE
Qm9 predict example with pickle

### PR DESCRIPTION
**Pull request by corochann**

### CODE REVIEW

**Comment by corochann** [comment]
Note that since this PR breaks the example code not to work on library on PyPI (version 0.2.0). New version should be released ASAP after this PR merged.

File: examples/qm9/train_qm9.py [hunk 1]
```diff
@@ -178,43 +189,37 @@ def main():
...
-
-    classifier = L.Classifier(model, lossfun=F.mean_squared_error,
-                              accfun=scaled_abs_error)
-
-    if args.gpu >= 0:
-        chainer.cuda.get_device_from_id(args.gpu).use()
-        classifier.to_gpu()
+    regressor = Regressor(
+        model, lossfun=F.mean_squared_error,
+        # metrics_fun={'abs_error': scaled_abs_error(scale=args.scale, ss=ss)},
```
Comment (mottodora) [inline examples/qm9/train_qm9.py]: please delete this line.

File: examples/qm9/train_qm9.py [hunk 2]
```diff
@@ -178,43 +189,37 @@ def main():
...
                                   'validation/main/loss',
-                                  'validation/main/accuracy',
+                                  'validation/main/abs_error',
                                   'elapsed_time']))
     trainer.extend(E.ProgressBar())
     trainer.run()
 
+    # --- save regressor & standardscaler ---
+    protocol = args.protocol
+    regressor.save_pickle(os.path.join(args.out, 'regressor.pkl'),
```
Comment (mottodora) [inline examples/qm9/train_qm9.py]: Please be possible to specify file name by argparser.

File: examples/qm9/predict_qm9.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        with open(os.path.join(args.in_dir, 'ss.pkl'), mode='rb') as f:
+            ss = pickle.load(f)
+    else:
+        ss = None
+
+    train_data_size = int(len(dataset) * train_data_ratio)
+    train, val = split_dataset_random(dataset, train_data_size, seed)
+
+    regressor = Regressor.load_pickle(
+        os.path.join(args.in_dir, 'regressor.pkl'),
```
Comment (mottodora) [inline examples/qm9/predict_qm9.py]: please be possible to specify file name.

**Comment by mottodora** [comment]
Can you resolve confilct?

File: examples/qm9/predict_qm9.py [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+import os
+import pickle
+
+from chainer.iterators import SerialIterator
+from chainer.training.extensions import Evaluator
+import pandas
+from sklearn.preprocessing import StandardScaler
+
+try:
+    import matplotlib
```
Comment (mottodora) [inline examples/qm9/predict_qm9.py]: I think `matplotlib` is not necessary.

**Comment by corochann** [comment]
resolved conflict, applied flake8

**Comment by delta2323** [comment]
Please add `predict_qm9.py` to the example test scripts (`example_test_cpu.sh` and `example_test_gpu.sh`)

**Comment by corochann** [comment]
I'd like to add test shell script in another PR.

**Comment by delta2323** [comment]
> I'd like to add test shell script in another PR.

OK

**Comment by delta2323** [comment]
As #139 is merged, could you merge or rebase the latest master branch?

**Pull request merged**